addOns/accessControl/src/main/javahelp/org/zaproxy/zap/extension/accessControl/resources/help/contents/concepts.html

```diff
@@ -69,6 +69,19 @@ <h3>Testing procedure</h3>
 </ul>
 <strong>Note:</strong> Access control testing is not allowed in <code>Safe</code> mode nor <code>Protected</code> if the context is not in scope.
 
+<H3>API</H3>
+
+The Addon exposes the following API enpoints:
+
+<ul>
+<li>accessControl.api.action.scan = Starts an Access Control scan with the given context ID and user ID. (Optional parameters: user ID for Unauthenticated 
```

Comment:
Was the key added on purpose? The WebSocket API help page shows just the name: https://github.com/zaproxy/zap-extensions/blob/master/addOns/websocket/src/main/javahelp/org/zaproxy/zap/extension/websocket/resources/help/contents/api.html